src/cli.rs

```diff
@@ -80,8 +84,7 @@ pub struct LoggingOpts {
     #[arg(long, default_value = "ghcid_ng=info")]
     pub tracing_filter: String,
 
-    /// How to display backtraces in error messages. '0' for no backtraces, '1' for standard
-    /// backtraces, and 'full' to display source snippets.
```

Comment:
Now that this argument is a `ValueEnum`, the possible values and their descriptions are displayed nicely in the `--help` output:

```
--backtrace <BACKTRACE>
    How to display backtraces in error messages

    [env: RUST_BACKTRACE=]
    [default: 0]

    Possible values:
    - 0:    Hide backtraces in errors
    - 1:    Display backtraces in errors
    - full: Display backtraces with all stack frames in errors
```